include/vsg/utils/TracyInstrumentation.h

```diff
@@ -12,12 +12,16 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
 
 </editor-fold> */
 
+#ifdef TRACY_ENABLE
```

Comment:
This header will only ever be included by applications that use Tracy so there is do point add TRACY_ENABLE, adding it inappropriate like this would break the build of their application if they actually used TRACY_ENABLE.

I think this is an example of trying to quieten false positives and breaking things in the process.

Reply:
The other parts of this file that use Tracy are already fenced off behind the same preprocessor check, so it's always required this define to be set to work, and never done anything productive when it's unset. I hadn't seen that Tracy's own headers disable most of their declarations when this is unset, and assumed it came from VSG.

It sounds like this file needs an exclusion if we're going to have a check in CI.